portal-api/src/v1/controllers/deal-eligibility-criteria.controller.js

```diff
@@ -3,10 +3,10 @@ const { userHasAccessTo } = require('../users/checks');
 const { getEligibilityErrors, getCriteria11Errors, getEligibilityStatus } = require('../validation/eligibility-criteria');
 const { getDocumentationErrors } = require('../validation/eligibility-documentation');
 const CONSTANTS = require('../../constants');
-const { findOneCountry } = require('./countries.controller');
+const { getCountry } = require('./countries.controller');
 
 const countryObject = async (countryCode) => {
-  const { data: countryObj } = await findOneCountry(countryCode);
+  const { data: countryObj } = await getCountry(countryCode);
```

Comment:
Just a heads up as this is now throwing an error - We're not handling this at this controller level so will return a 500 if this occurs (https://expressjs.com/en/guide/error-handling.html)

Reply:
Have refactored.